Cargo.lock

```diff
@@ -99,24 +108,24 @@ dependencies = [
 
 [[package]]
 name = "anyhow"
-version = "1.0.51"
+version = "1.0.65"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "8b26702f315f53b6071259e15dd9d64528213b44d61de1ec926eca7715d62203"
+checksum = "98161a4e3e2184da77bb14f02184cdd111e83bbbcc9979dfee3c44b9a85f5602"
 
 [[package]]
 name = "approx"
-version = "0.5.0"
+version = "0.5.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "072df7202e63b127ab55acfe16ce97013d5b97bf160489336d3f1840fd78e99e"
+checksum = "cab112f0a86d568ea0e627cc1d6be74a1e9cd55214684db5561995f6dad897c6"
 dependencies = [
  "num-traits",
 ]
 
 [[package]]
 name = "arbitrary"
-version = "1.0.3"
+version = "1.1.6"
```

Comment:
New version of arbitrary bumped MSRV to 1.63. Our CI doesn't have it yet.

It looks like you did `cargo update` which we strongly discourage in favor of reviewing individual dependabot updates that are actually reviewed.

I'd suggest reverting all of those changes 
```
cargo checkout master Cargo.lock
```

Reply:
Looks like something in fe65488d6092130040d7a9008663c11b8ed7ca2e1 went wrong while merging.